.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: lint
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run pre-commit test
+      run: |
+        curl https://pre-commit.com/install-local.py | python3 -
+        curl -L "$(curl -s https://api.github.com/repos/segmentio/terraform-docs/releases/latest | grep -o -E "https://.+?-linux-amd64")" > terraform-docs && chmod +x terraform-docs && sudo mv terraform-docs /usr/bin/
```

Comment:
Maybe use `jq` to select the value from json?
My gut reaction is that there may be several lines that match `-linux-amd64` and then it's not clear what the script will download...

Reply:
It's written like that in the official documentation to install